framework/flamingo/log.go

```diff
@@ -67,27 +67,63 @@ var _ Logger = new(StdLogger)
 
 // StdLogger uses the go stdlib logger for logging
 type StdLogger struct {
-	log.Logger
+	Logger log.Logger
+}
+
+func isLoggerNil(l *StdLogger) bool {
+	return l == nil || l.Logger.Writer() == nil
+}
+
+func (l *StdLogger) Fatal(args ...interface{}) {
+	if isLoggerNil(l) {
+		log.Fatalln(args...)
+		return
+	}
+	l.Logger.Fatal(args...)
```

Comment:
The linter wants a newline between `}` and `l.Logger.Fatal...`
https://github.com/bombsimon/wsl/blob/master/doc/rules.md#recommended-amendment-9